zebra/main.c

```diff
@@ -77,6 +77,9 @@ int allow_delete = 0;
 /* Don't delete kernel route. */
 int keep_kernel_mode = 0;
 
+/* Perform kernel reconciliation */
+int kernel_reconcile = 0;
```

Comment:
Let's call this graceful restart.  this variable belongs in zebra_router.c and .h as well.  It does not belong to main.c

Reply:
This variable say "kernel_graceful_restart" (instead of kernel_reconcile) will be set with zebra option -K (--kernel-graceful), So I think main.c is good place for it.

I am fine to keep it in zebra_router.c and .h if you still feel that is right place for option variable. Thanks Donald.

Reply:
Added in zebra_router.c and .h as of now.